app/src/org/commcare/dalvik/dialogs/AlertDialogFactory.java

```diff
@@ -0,0 +1,143 @@
+package org.commcare.dalvik.dialogs;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.commcare.dalvik.R;
+import org.javarosa.core.services.locale.Localization;
+
+/**
+ * Created by amstone326 on 10/9/15.
+ */
+public class AlertDialogFactory {
+
+    private AlertDialog dialog;
+    private View view;
```

Comment:
Ah yea I don't :/ I tried setting them up once, ran into some road bumps, and didn't try again.. Will make sure to figure it out though.